Formula/distrobox.rb

```diff
@@ -16,6 +16,16 @@ def install
     system "./install", "--prefix", prefix
   end
 
+  def caveats
+    <<~EOS
+      Note: Distrobox requires one of podman or docker.
+      You can install either using `brew` or follow the documentation on:
+      	man distrobox-compatibility
+      or:
+      	https://github.com/89luca89/distrobox/blob/main/docs/compatibility.md
```

Comment:
```suggestion
      Distrobox requires one of podman or docker. Do
        brew install podman
      or consult the documentation for details.
```
Let's keep `caveats` as minimal as possible to avoid having users ignore them (because they get too long).